components/symbl_ai/symbl_ai.app.mjs

```diff
@@ -24,6 +25,31 @@ export default {
         }));
       },
     },
+    memberId: {
+      type: "string",
+      label: "Member Id",
+      description: "The unique identifier of the member in the Conversation.",
+      async options({
+        page, conversationId,
+      }) {
+        const limit = 50;
+        const params = {
+          limit,
+          offset: limit * page,
+          order: "desc",
+        };
+        const { members } = await this.getMembers({
+          params,
+          conversationId: get(conversationId, "value", conversationId),
```

Comment:
I'm a bit confused with this assigment, is this necessary? why don't you just do
```suggestion
          conversationId,
```